wrappers/CMakeLists.txt

```diff
@@ -111,8 +112,8 @@ set(PEGTL_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src/object-store/external/pegt
 add_subdirectory(src/object-store/src)
 add_subdirectory(src)
 
-if(APPLE)
-  # force Object Store to use the generic EventLoopSignal implementation instead of the NSRunLoop-one
+if(APPLE OR ANDROID)
```

Comment:
On Windows and Linux it already picks up the generic EventLoopSignal. We’d get linker errors if we compiled the file twice.